independent-publisher-connectors/Philips HUE/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2176 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Philips HUE",
+    "description": "Philips HUE allows you to control smart home devices, like lights, switches, plugs and more built by Philips.",
+    "version": "1.0",
+    "contact": {
+      "name": "Tomasz Poszytek",
+      "url": "https://aka.ms/poszytek",
+      "email": "tomasz@poszytek.eu"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.philips-hue.com/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.philips-hue.com/en-us/support/legal/privacy-policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Internet of Things;Lifestyle and Entertainment"
+    }
+  ],  
+  "host": "api.meethue.com",
+  "basePath": "/route",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/clip/v2/resource/light": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "errors": {
+                  "type": "array",
+                  "items": {
+                    "description": "List of errors.",
+                    "type": "object",
+                    "properties": {
+                      "description": {
+                        "type": "string",
+                        "x-nullable": true,
+                        "description": "A human-readable explanation specific to this occurrence of the problem.",
+                        "title": "Error description"
+                      }
+                    }
+                  },
+                  "description": "Errors that occured during the call."
```

Comment:
All descriptions in the apiDefinition.swagger.json file should be free from grammatical errors. This also applies to summaries, titles, etc. Please review them and fix them.